Automattic/wp-calypso
# Move theme-related selectors to `state/themes`

This PR moves the remaining theme-related selectors to `state/themes/selectors`, now that theme state is modularised (#39376), and the `state/themes/selectors` monolith has been broken down into individual selectors (#39605).

There are no code changes other than trivial lint fixes and updating one test to use `expect` instead of `assert` (also a lint fix, albeit a more elaborate one).

#### Changes proposed in this Pull Request

* Move theme-related selectors in `state/selectors` to `state/themes/selectors`

#### Testing instructions

There are no functionality code changes, so it should be sufficient to ensure that the build works and existing unit tests don't fail.
